### DISCUSSION TITLE
is there a way to select non-consecutive cells?

### DISCUSSION THREAD

**Discussion by dyf-bits** [andmarti1424/sc-im#929] [General]
visual mode allows selecting cells that sit beside each other. is there a way to select non-consecutive cells?

**Comment by andmarti1424** [comment]
Hello. I am afraid no. What would you do after selecting them?
Delete the content?

**Comment by dyf-bits** [comment]
I believe I wanted to change their colors and make them stand out

**Reply by martinengelke** [reply]
peu à peu
